checks/debian_hosts.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+begin
```

Comment:
Copied over from hostname.rb, I think that's because Ruby 1.8 needs that ```require 'rubygems'``` explicitly.